_config.yml

```diff
@@ -122,6 +125,10 @@ defaults:
       path: 'docs/login-gov-press-kit.pdf'
     values:
       sitemap: false
+  - scope:
+      path: 'partners/program-updates/'
+    values:
+      sitemap: false
```

Comment:
I'm assuming we'll want to remove this in the future once we have some articles?

Reply:
@aduth Yes, that's correct. Thank you for the reminder.